### PR TITLE
feature: Added formatted uptime value for alert templates #4983

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Fixes: #4983

### CODE REVIEW

File: doc/Extensions/Alerting.md [hunk 1]
```diff
@@ -118,6 +118,9 @@ Placeholders:
 - Hostname of the Device: `%hostname`
 - sysName of the Device: `%sysName`
 - location of the Device: `%location`
+- uptime of the Device (in seconds): `%uptime`
+- short uptime of the Device (28d 22h 30m 7s): `%uptime`
```
Comment (Rosiak) [inline doc/Extensions/Alerting.md]: Typo? :)

File: doc/Extensions/Alerting.md [hunk 2]
```diff
@@ -118,6 +118,9 @@ Placeholders:
...
+- long uptime of the Device (28 days, 22h 30m 7s): `%uptime`
```
Comment (Rosiak) [inline doc/Extensions/Alerting.md]: Typo? :)

File: alerts.php [hunk 3]
```diff
@@ -449,14 +449,16 @@ function DescribeAlert($alert)
...
-    $obj['description'] = $device['purpose'];
-    $obj['notes']       = $device['notes'];
-    $obj['device_id']   = $alert['device_id'];
-    $extra              = $alert['details'];
+    $obj['hostname']     = $device['hostname'];
+    $obj['sysName']      = $device['sysName'];
+    $obj['location']     = $device['location'];
+    $obj['uptime']       = $device['uptime'];
+    $obj['uptime_short'] = formatUptime($device['uptime'], 'short');
+    $obj['uptime_long'] = formatUptime($device['uptime']);
```
Comment (Rosiak) [inline alerts.php]: I'm being very picky, but could you add a single space so it aligns with the other lines? :)

**Comment by laf** [comment]
Done.

I've kind of stopped aligning things all the time as one longer line after screws it all up :(

**Comment by Rosiak** [comment]
Did you commit your updates?

**Comment by laf** [comment]
Hadn't pushed for some reason.

**Pull request merged**